### DISCUSSION TITLE
Angular 15

### DISCUSSION THREAD

**Discussion by gabriiels** [the-guild-org/apollo-angular#1853] [General]
Apollo-angular for Angular version 15

**Comment by 53845714nF** [comment]
It looks like there are ready commits by @PowerKiKi  that support angular 15. But I also had the problem that it didn't work. 
How is the release cycle?

**Reply by gabriiels** [reply]
Temporary solution: 

https://github.com/kamilkisiela/apollo-angular/issues/1847#issuecomment-1323762494

With Angular Schematics it does not work but with installation without schema it works.
https://the-guild.dev/graphql/apollo-angular/docs/get-started#installation-without-angular-schematics

I don't know how to handle the release cycle.

**Reply by PowerKiKi** [reply]
Only @kamilkisiela can cut releases. And he does more or less whenever there is something to release, so hopefully it will be soon-ish

**Reply by Aaditya-Rane** [reply]
Any update on this? any idea when this will be released?
